modules/gdscript/tests/scripts/parser/features/multiline_assert.gd

```diff
@@ -0,0 +1,24 @@
+func test():
+	var x := 5
+
+	assert(x > 0)
+	assert(x > 0,)
+	assert(x > 0, 'message')
+	assert(x > 0, 'message',)
+
+	assert(
+		x > 0
+	)
+	assert(
+		x > 0,
+	)
+	assert(
+		x > 0,
+		'message'
+	)
+	assert(
+		x > 0,
+		'message',
+	)
```

Comment:
Why are the trailing commas allowed? It doesn't make any sense to me, I'd expect an error when there's a trailing comma.

Reply:
I agree, the trailing comma doesn't really make sense.

Reply:
I'm not sure I understand the surprise:
```gdscript
func method(a, b = 'c') -> void:
  print(a, b)

func _ready() -> void:
  method('a',)
  method('a', 'b',)
  print(['a',])
```
Trailing commas and multi-line are supported by method calls and arrays. That's just make it behave like consistent with those. What am I missing?

Reply:
@vnen Is this a bug or a feature?

Reply:
It is a feature. Search for "// Allow for trailing comma." comment in parser - `parse_call`, `parse_signal`, `parse_enum`, `parse_function_signature`, `parse_array`, `parse_dictionary` all have that comment for code that allows trailing comma.